### ISSUE TITLE
How to get JetPack on the duckiebot (DB21M) ?

### ISSUE THREAD

**Issue by bishoyroufael** [duckietown/dt-machine-learning-libraries#4]
I am getting confused about how to setup a proper AI environment onto my DB21M. I don't see any NVIDIA drivers when SSHing into the bot or any CUDA installed. How to install JetPack WITHOUT re-flashing it with the Jetson OS as denoted on NVIDIA documentation? I want to use it with ROS.

**Comment by claganier-turing** [comment]
@bishoyroufael Did you ever solve this ?

**Comment by bishoyroufael** [comment]
No. I ended up doing AI training on my laptop machine with communication being sent bi-directional over a local network via an API. _(slow and terrible solution but it was for a PoC to a thesis project)_
